tests/frontend/test_kibana_app/cypress/support/index.js

```diff
@@ -21,6 +21,7 @@ import { updateCookies, clearSession, updateExpiryValueCookies } from '../integr
 const cookieMock = require('../../cookie.json');
 const loginMethod = 'xpack'
 import './commands';
+require("cypress-xpath");
 
 // Alternatively you can use CommonJS syntax:
```

Comment:
are these comments needed?.

Reply:
I know you didn't add them. but they are not very helpful

Reply:
You are right, I will ask @mauceballos if any of these comments are important's to understand the cookie storage in order to delete the unnecessary comments.